src/rockstor/storageadmin/views/pool.py

```diff
@@ -51,6 +53,54 @@ def _validate_disk(d, request):
             handle_exception(Exception(e_msg), request)
 
     @staticmethod
+    def _get_disk_names(disks, request):
```

Comment:
this method is not being called anywhere. Can it be removed? You can also consider list comprehension for creating lists like this.